vars/helm.groovy

```diff
@@ -1,12 +1,41 @@
+def getExtraCommands(chartName, tag) {
```

Comment:
`chartName` doesn't look to be used within this function - can it be removed?

Reply:
Removed